dbt-cta/snapchat_marketing/models/0_ctes/adsquads_targeting_ab3.sql

```diff
@@ -6,10 +6,7 @@
 -- depends_on: {{ ref('adsquads_targeting_ab2') }}
 select
     {{ dbt_utils.surrogate_key([
```

Comment:
@emily-flambe I'm not using this surrogate key any more (the primary key of the table should be `ad_squad_id`). I kept this bit in for continuity's sake, but would it it be simpler if I just dropped the SK and the model entirely?

Reply:
ah! if we aren't using it then no need to keep the _ab3 step - just need to make sure any downstream reference(s) to `_ab3` are replaced with `_ab2`. In general if we can use a UPK other than the hashid that Airbyte generates in `_ab3`, we should preferentially use that instead and yeet `_ab3` - we just haven't taken the time to be super rigorous about that 🫠🫠🫠🫠

Reply:
roger that! I will do that for all of these objects then, a bunch of these "simpler" models just have a straight `id` field we can use. Will increase the scope of this refactor a bit but I think it's worth it.